site/pages/join-us/index.js

```diff
@@ -103,46 +80,84 @@ const apply = {
 const vacancies = {
   type: 'Title2',
   props: {
-    children: 'Current Vacancies',
+    children: <span className={styles.vacancyHeader}>Current Vacancies</span>,
   },
 };
 
-export default function JoinUs({ jobs }) {
-  return (
-    <div className={styles.background}>
+class HeaderContainer extends React.Component {
+  static showPicture() {
+    const MEDIUM_BREAKPOINT = '(min-width: 690px)';
+    return !window.matchMedia(MEDIUM_BREAKPOINT).matches;
+  }
+
+  state = {
+    show: HeaderContainer.showPicture(),
+  };
```

Comment:
For initial state to be set correctly, you need to use a constructor:

```
constructor() {
  super();
  this.state = { show: ... }
}
```

Reply:
https://reactjs.org/docs/react-component.html#constructor

Reply:
happy to do this but i thought this way was ok?  
https://github.com/Khan/style-guides/blob/master/style/react.md#use-es2015-classes
https://stackoverflow.com/questions/35662932/react-constructor-es6-vs-es7
I am probably missing something though, can you elaborate please?

Reply:
did not know that will work! all good 👍